docs/xss.rst

```diff
@@ -0,0 +1,95 @@
+Performing XSS Vulnerability Audits
```

Comment:
@catong can you please help me by reviewing this doc? It's pretty minimal at the moment (and the example case is a bit contrived-- you can see more realistic usages in https://github.com/edx/edx-platform/pull/11758).